regression/pageObjects/homepage.js

```diff
@@ -0,0 +1,62 @@
+class homepage {
+    
+    async clickFashion() {
+        let fashion = element.all(by.linkText('Fashion'));
+        await fashion.click();
+    }
+    async clickAccessories(){
```

Comment:
is it possible to add perhaps :
```
async clickAccessories(filed_name){
let accessories = element.all(by.xpath('//a[normalize-space()="#{filed_name}"]'));
         await accessories.click();
     }

```
so that you don't need to write all those locators you just pass
clickAccessories('Accessories') , clickAccessories('Electronics') etc